### ISSUE TITLE
SendHttpRequest Activity v3 Response not being captured 

### ISSUE THREAD

**Issue by sagargolu** [elsa-workflows/elsa-core#3839]
We were previously able to get/store in variable response of the HttpRequest activity in v3 version but now we are not able to get it any anyway. We were storing that in variable like attached image.

![image](https://user-images.githubusercontent.com/79979623/227948254-0145a2da-f245-4354-b80e-4a43b727cf2f.png)
 
We are responding back with HttpResponse but some how we are getting null for both the variable.

![image](https://user-images.githubusercontent.com/79979623/227948400-40f7df2b-fb95-41c8-9bbf-a314a55f18e0.png)

**Comment by sfmskywalker** [comment]
Any chance you can export the workflow for me to investigate?

**Comment by sagargolu** [comment]
[search.zip](https://github.com/elsa-workflows/elsa-core/files/11086167/search.zip)

PFA.... I have created dummy response from webhook.site after post any data to the site.

{
"ResponseStatus":"Received"
}

**Comment by sagargolu** [comment]
@sfmskywalker any update on this?

**Comment by sagargolu** [comment]
I have reviewed in debug mode. So can you check if this is making an issue? Because system is able to capture response and able to parse but it is not able to store it or capture it within variable.

![image](https://user-images.githubusercontent.com/79979623/228811204-f88cbdbb-35cd-4c20-a9bc-aac78cf07999.png)

**Comment by sfmskywalker** [comment]
@sagargolu I just tried with the latest version of the source code and your workflow JSON.
In order for it to work, I had to first associate the variables with the outputs of the HTTP Endpoint and Send HTTP Request activities. After that, it worked well.

This is the request I sent to the workflow:

```bash
curl --location 'https://localhost:5001/workflows/case1' \
--header 'Content-Type: application/json' \
--data '{
    "prop1": 5
}'
```

And this was the response:

```json
{
    "output": {
        "ResponseStatus": "Received"
    },
    "Res": {
        "prop1": 5
    }
}
```

I noticed that the Content field of the HTTP Request activity is stringifyied from a JS object into JSON. Athough this is absolutely fine, you don't have to if you prefer to just return the JavaScript object. The activity will recognize that the returned object is a JavaScript object (represented as an ExpandoObject in .NET) and will be serialized to a JSON string for you.
Neither approach is better, but I thought it might be useful to know about this.

I attached an export of my version of the workflow that you might try out.

[workflow-definition-2001cf3c57da47e9a5bb38d3382cb55a (1).json.zip](https://github.com/elsa-workflows/elsa-core/files/11111823/workflow-definition-2001cf3c57da47e9a5bb38d3382cb55a.1.json.zip)
